bigtable-dataflow-parent/bigtable-dataflow-import/pom.xml

```diff
@@ -124,7 +124,6 @@ limitations under the License.
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-eclipse-plugin</artifactId>
-                <version>2.9</version>
```

Comment:
It was removed from the child, and now the version is managed in the top pom.xml for all usages of maven-eclipse-plugin